### PR TITLE
Cookie banner added

### PR DESCRIPTION
**Pull request by pcoenen**

### CODE REVIEW

File: views/home.php [hunk 1]
```diff
@@ -203,5 +203,24 @@ function showSelectedCourses(){
...
+			    "popup": {
+			      "background": "#343a40"
+			    },
+			    "button": {
+			      "background": "#007bff"
+			    }
+			  },
+			  "showLink": false,
+			  "content": {
+			    "message": "This website uses cookies to personalise content and to analyse our traffic."
```
Comment (MidasLamb) [inline views/home.php]: "Analyse our traffic" should be removed.

**Review by MidasLamb — CHANGES_REQUESTED** [review]

**Comment by pcoenen** [comment]
Changed

**Comment by MidasLamb** [comment]
image
Cookie banner messes with the existing cookies after accepting it (see url on the right of the screenshot). Maybe there is an option for the cookiebanner to put it's save-token elsewhere? Localstorage maybe? 

The bug occurs after selecting courses and accepting the cookie, and then refreshing twice.

**Comment by pcoenen** [comment]
I put the courses in local storage

**Review by MidasLamb — APPROVED** [review]
I'm just assuming it works now

**Comment by MidasLamb** [comment]
Unfortunately, this is not backwards compatible.

**Pull request merged**